Tests/SwiftLintFrameworkTests/BaselineTests.swift

```diff
@@ -0,0 +1,225 @@
+@testable import SwiftLintBuiltInRules
+@testable import SwiftLintCore
+import XCTest
+
+private var temporaryFilePath: String {
```

Comment:
Let this only be a (fixed?) folder
```suggestion
private var temporaryFolderPath: String {
```
and take care of changing the active directory there in `setUp`; switch back in `tearDown`. Have `withExampleFileCreated` create the example and manage its removal.